fieldservice/wizard/fsm_wizard.py

```diff
@@ -31,7 +31,7 @@ def action_convert(self):
     def _prepare_fsm_location(self, partner):
         return {
             'partner_id': partner.id,
-            'owner_id': partner.id,
+            'owner_id': partner.commercial_partner_id.id
```

Comment:
Iv've seen cases where the we have multiple levels here. I would rather use `parent_id`, if possible.
```suggestion
            'owner_id': partner.parent_id.id
```

Reply:
What is the hierarchy of partners you have ?
I don't think odoo supports a multi level : a company can't really be a child of another company without side effects.